From 15ed265a84847a29b5c13f00ce51feb74c63a427 Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Sun, 28 Feb 2016 00:16:03 +0300
Subject: [PATCH 7/7] [TRANSLATION] Irkutsk is not Yakutsk

Fix timezone mistake about Irkutsk and Yakutsk in Russian translation
Irkutsk is in timezone GMT+8, Yakutsk in timezone GMT+9

Change-Id: Ice1018d31a1e79a0dbb21ce5ff6b8b86b835742c
---
 gui/theme/common/languages/en.xml | 2 +-
 gui/theme/common/languages/ru.xml | 6 +++---
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/gui/theme/common/languages/en.xml b/gui/theme/common/languages/en.xml
index 4b7743a..4bdd851 100644
--- a/gui/theme/common/languages/en.xml
+++ b/gui/theme/common/languages/en.xml
@@ -328,7 +328,7 @@
 		<string name="utcp5">(UTC +5) Yekaterinburg, Islamabad</string>
 		<string name="utcp6">(UTC +6) Almaty, Dhaka, Colombo</string>
 		<string name="utcp7">(UTC +7) Bangkok, Hanoi, Jakarta</string>
-		<string name="utcp8">(UTC +8) Beijing, Singapore, Hong Kong</string>
+		<string name="utcp8">(UTC +8) Beijing, Singapore, Hong Kong, Irkutsk</string>
 		<string name="utcp9">(UTC +9) Tokyo, Seoul, Yakutsk</string>
 		<string name="utcp10">(UTC +10) Eastern Australia, Guam</string>
 		<string name="utcp11">(UTC +11) Vladivostok, Solomon Islands</string>
diff --git a/gui/theme/common/languages/ru.xml b/gui/theme/common/languages/ru.xml
index f1dd011..fa73f98 100755
--- a/gui/theme/common/languages/ru.xml
+++ b/gui/theme/common/languages/ru.xml
@@ -329,9 +329,9 @@
 		<string name="utcp5">(UTC +5) Ташкент, Исламабад, Карачи</string>
 		<string name="utcp6">(UTC +6) Алматы, Астана, Дхака, Екатеринбург</string>
 		<string name="utcp7">(UTC +7) Омск, Новосибирск, Бангкок, Джакарта, Ханой</string>
-		<string name="utcp8">(UTC +8) Красноярск, Гонконг, Пекин, Улан-Батор</string>
-		<string name="utcp9">(UTC +9) Иркутск, Осака, Саппоро, Токио, Сеул</string>
-		<string name="utcp10">(UTC +10) Якутск, Гуам, Канберра, Мельбурн, Сидней</string>
+		<string name="utcp8">(UTC +8) Красноярск, Гонконг, Пекин, Улан-Батор, Иркутск</string>
+		<string name="utcp9">(UTC +9) Осака, Саппоро, Токио, Сеул, Якутск</string>
+		<string name="utcp10">(UTC +10) Гуам, Канберра, Мельбурн, Сидней</string>
 		<string name="utcp11">(UTC +11) Владивосток, Сахалин, Соломоновы о-ва</string>
 		<string name="utcp12">(UTC +12) Магадан, Камчатка, Фиджи, Маршаловы о-ва</string>
 		<string name="sel_tz_offset">Выбор смещения (как правило 0): %tw_time_zone_guioffset%</string>
-- 
2.5.0

